docs/releasing.md

```diff
@@ -42,6 +42,32 @@ git push origin "${RELEASE_VERSION}"
 * If this is a pre-release tick `This is a pre-release`
 * Publish the draft release and when asked say yes to creating a discussion.
 
+## Commit a new `buf` tag
+
+We have gRPC API docs hosted on [buf.build](https://buf.build/weaveworks-liquidmetal/flintlock).
+If the API has changed, you'll need to update these.
+
+Log in creds can be found in the shared Team Quicksilver 1Pass vault.
```

Comment:
It's a bit weird here, on an open source project.

Reply:
yeh true. but i couldn't think where to put that info and i didn't want it to just be me with the access 🤔